README.md

```diff
@@ -45,13 +45,54 @@ See the issues for things to work on.
 * Uses OS distribution methods for software when possible
 
   Use `apt` or `homebrew` or similar.
-  
+
 * Avoid using PPAs where possible.
 
   PPAs can be useful for grouping more different versions of software together, beyond what is distributed by the base OS, but they tend not to be supported forever, and aren't automatically tested for compatibility as new OS versions come out.  Also, base OS support of the components used here is much better than it was.
 
 ## Contribute
-The webpage is built using [GitHub Pages](https://pages.github.com/). To compile your changes, execute
-* `bundle install --path .vendor/bundle`
-* `bundle exec jekyll serve`
-and open http://127.0.0.1:4000/ in the browser
+
+This is a re-work of original <https://switch2osm.org> website using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+
+The goal is to have instructions in different languages in one place.
+
+You are welcome to extend the styling and content, add new languages. To do this, clone this repository, install the necessary dependencies and experiment.
+
+```
+git clone git@github.com:switch2osm/switch2osm.git
+cd switch2osm
+python -m venv venv
+source ./venv/bin/activate
+pip install -r requirements.txt
+mkdocs serve
+```
+
+Open <http://127.0.0.1:8000/switch2osm/> in your browser.
+
+## Translation
+
+![](assets/help-to-translate.png)
+
+<!-- Translation into other languages is carried out on [Transifex](https://app.transifex.com/osm-ua/switch2osm/dashboard/). Click "Help to translate" in the language switcher to get there. -->
```

Comment:
@pnorman, if the mention of `osm-ua` here confuses you, feel free to provide another Transifex organization name to replace it. In any case, this line is commented out and can become active immediately after establishing the translation routines on Transifex.

Reply:
I've tried creating `openstreetmap/switch2osm` but since osm-ua has already taken `switch2osm` I can't.

Reply:
It looks like a Transifex bug 🤷‍♂️ 

I've chnaged the name for the project in osm-ua org.
<img width="294" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/290eae7d-2c16-46d2-a57f-5867c2345dd4">

@grischard could you try to create the project again?

Reply:
It still says "Project with this Slug already exists." Did you change just the name or also the slug?

<img width="720" alt="image" src="https://github.com/switch2osm/switch2osm/assets/1556567/2c1c574a-5015-451a-9e65-20b4322f527c">

Reply:
🤕 Try one more time. I've just changed slag as well
<img width="597" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/3fbaea90-4cc1-49c6-a081-c3cbb7004ccf">

Reply:
https://app.transifex.com/openstreetmap/switch2osm/dashboard/ is up.

Reply:
This strange behavior is 100% Transifex's bug

Reply:
<img width="741" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/fae931ad-cca4-45f3-b821-eb924f4fc7e9">
@grischard Now is time to add GitHub integration and use `transifex.yaml` in it

Reply:
<img width="745" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/eb32f28d-245d-4529-a3c2-617a4ea17355">

<img width="745" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/62aa6b30-2763-4639-aab2-6c1641aa84a7">

Reply:
<img width="745" alt="image" src="https://github.com/switch2osm/switch2osm/assets/369696/8b30642e-037b-4a9d-9375-bf2280bc7f24">
Then add custom placeholders to preserve specific markdown formating

Reply:
I mark this conversation as resolved. 

Thanks, @grischard, for creating a project on Transifex for further translation work.